### ISSUE TITLE
Licence, citation and reuse ?

### ISSUE THREAD

**Issue by roya4m** [observingClouds/eurec4a_snd#25]
I would like to reuse this code to convert radiosondes from the same different platforms (Vaisala, MeteoModem) for a field experiment called SOFOG3D (SOuth westFOGs 3D experiment for processes study) conducted by Meteo France/CNRM on fog properties in France during last winter. 

Maybe a GNU/GPL license is a good one: this allows the use of the code (all-purpose), its modification and the re-publication of the modified code. If the code is modified, the source must be cited and the modifications must be listed.

For know I planned to cite the software in the NetCDF file in the attribute created_with, for example created_with = "L1_mwx.py 10.5281/zenodo.4391098" with adding the Licence. 

Thanks a lot for all the work already done!

**Comment by observingClouds** [comment]
Hi @roya4m, great to hear that this code is also useful for others. I used added the GPL license to the repository and the code published on zenodo.
Be aware, that some scripts are quite specific to the EUREC4A data and correct for some hick-ups we introduced during the campaign. As mentioned in #24 I'm planning to publish a more general package that can more easily be reused ( in ~ two months) and is better documented. The `L1_mwx.py` should be free of these specific though.
